### DISCUSSION TITLE
Unleash GO SDK Version in docs site

### DISCUSSION THREAD

**Discussion by xico42** [Unleash/unleash#8829] [General]
In the [unleash documentation site](https://docs.getunleash.io/reference/sdks/go#1-install-unleash-client-go) regarding the golang SDK it recommends installing the version 3 of the sdk, even though the v4 was released in [november of 2023](https://github.com/Unleash/unleash-client-go/releases/tag/v4.0.0).

```
go get github.com/Unleash/unleash-client-go/v3
```

Is this intended?

**Comment by gastonfournier** [comment]
Hi @xico42! I think it was just an overlook when we released the new version. This PR should fix it: https://github.com/Unleash/unleash/pull/8857 once merged
